ubuntu/httpdir/user-data

```diff
@@ -14,8 +14,46 @@ autoinstall:
   ssh:
     install-server: true
   storage:
-    layout:
-      name: direct
```

Comment:
Just remembered that I looked at this back in the day: https://github.com/NeCTAR-RC/nectar-images/blob/master/http/ubuntu-22.04/user-data#L18

Maybe we should read carefully how they manage on that repo